src/modules/overview/overview.vue

```diff
@@ -345,22 +360,32 @@ export default Vue.extend<
                 resolvedVehicleCategories[category] = { color, vehicles: {} };
                 Object.entries(groups).forEach(
                     ([group, vehicles]) =>
+                        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                        // @ts-ignore
```

Comment:
why?

Reply:
because typescript has been confused by the code and I was not able to find a way to unconfuse Typescript 😅